remix/app/repositories/newsletter.server.test.ts

```diff
@@ -27,14 +27,14 @@ describe('Newsletter Repository', () => {
     });
   });
 
-  describe('findOne', () => {
+  describe('findFirst', () => {
```

Comment:
Why the change from `findOne` to `findFirst`?

Reply:
For this, I was thinking to name the same `Prisma` method and try to make it more meaningful, but thinking it again they both return a similar result. So I change it back now.